src/simple-cache-client.ts

```diff
@@ -232,9 +239,12 @@ export class SimpleCacheClient {
   }
 
   /**
-   * Get the number of values in a list.
-   * @param {string} cacheName - Name of the cache with the list.
+   * Gets the number of elements in the given list.
+   *
+   * @param {string} cacheName - The cache containing the list.
    * @param {string} listName - The list to get the length of.
+   * @returns {Promise<CacheListLength.Response>} - The result of the get
+   * operation (hit, miss, or error) and the list length.
```

Comment:
The wording here trips me up a bit since the list length is only part of a hit response. Can we rephrase this to make that clear? (Same goes for other operations that carry extra information in the hit/success)